python/cudnn/gemm/cutedsl/grouped/glu/api.py

```
@@ -69,9 +100,9 @@ class GluCall:
     b_major: str = "k"
     norm_const_tensor: Optional[torch.Tensor] = None
     prob_tensor: Optional[torch.Tensor] = None
-    acc_dtype: torch.dtype = torch.float32
-    c_dtype: torch.dtype = torch.bfloat16
-    d_dtype: torch.dtype = torch.bfloat16
+    acc_dtype: Optional[torch.dtype] = None
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)